### ISSUE TITLE
Trying to update my game engine from Chaiscript 5.8.x to 6.x.x

### ISSUE THREAD

**Issue by define-private-public** [ChaiScript/ChaiScript#363]
I haven't done any work on [Masala](https://gitlab.com/define-private-public/Masala) in about a year in a half, but I wanted to see if there were any performance boosts if I updated the ChaiScript version it uses.

I saw in the release notes that there was a shift in how the ChaiScript VM is instantiated.  I thought I was doing it right:
https://gitlab.com/define-private-public/Masala/blob/chaiscript-6.0.0/src/masalaapp.cpp#L31

But when I launched `masala.bin`, I got this error:
```
terminate called after throwing an instance of 'chaiscript::exception::eval_error'
  what():  Error: "Can not find object: use" 
```

What's going on?

**Comment by lefticus** [comment]
You hit an issue that was fixed in 36e61dec0a1412ba2a09e15c6f45785f24915daf you can either update to the latest `develop` or you can explicitly pass the default arguments as we see here: https://github.com/ChaiScript/ChaiScript/commit/36e61dec0a1412ba2a09e15c6f45785f24915daf#diff-443e50d0c572da53c08a0921728c6cb4L835

**Comment by define-private-public** [comment]
Okay, this works.  thanks!
